test/html.jl

```diff
@@ -73,6 +74,7 @@
             <link rel="stylesheet" href="/Books.jl/files/style.css"/>
             <a href="/Books.jl/test.html#sec:foo">Foo</a>
             <h2 data-number="3.5" id="sec:foo"><span class="header-section-number">3.5</span> Foo</h2>
+            <p> <a href="https://www.google.com">Jump</a></p>
```

Comment:
This line will become the following before this PR:

```
<a href=\"/Books.jlhttps://www.google.com\">Jump</a></p>
```

@rikhuijzer

Reply:
Now I get it! Good addition and nice that you've added a test too :smile: :100: